src-frontend/app/components/page-list.js

```diff
@@ -13,8 +12,8 @@ export default Ember.Component.extend({
             update: function(event, ui) {
                 var indices = {};
 
-                $(this).find('a.open-page').each(function(index) {
-                    indices[$(this).data('id')] = index;
+                Ember.$(this).find('a.open-page').each(function(index) {
```

Comment:
Yup, and for performance reasons you're supposed to use it instead of the global one.